server.py

```diff
@@ -56,12 +62,23 @@ async def delete_url(alias: str):
     if(sqlite_helpers.delete_url(DATABASE_FILE, alias)):
         return {"message": "URL deleted successfully"}
     else:
+        logging.error("url not found")
         raise HTTPException(status_code=HttpResponse.NOT_FOUND.code)
 
 @app.exception_handler(HTTPException)
 async def http_exception_handler(request, exc):
     status_code_enum = http_code_to_enum[exc.status_code]
     return HTMLResponse(content=status_code_enum.content, status_code=status_code_enum.code)
 
+logging.Formatter.converter = time.gmtime
+
+logging.basicConfig(
+    # in mondo we trust
+    format="%(asctime)s.%(msecs)03dZ %(levelname)s:%(name)s:%(message)s",
+    datefmt="%Y-%m-%dT%H:%M:%S",
+    level= logging.ERROR - (args.verbose*10),
+)
+
 if __name__ == "__main__":
+    logging.info(f"running on {args.host}, listening on port {args.port}")
```

Comment:
can you check if running the server it already prints the host/port it listens on? if so we can remove
```suggestion
```